resources/tests/conftest.py

```diff
@@ -64,14 +64,17 @@ def winter_storage_place():
 
 
 @pytest.fixture
-def user():
-    user = UserFactory()
-    return user
-
+def user(request):
+    permissions = request.param
```

Comment:
what does pytest pass here, when the fixture is called in a regular way, i.e. not from a parametrize call? I mean, what is the default for this `request.param`? do we need to have a default `else` for when it is `None`? 🤔

Reply:
When you just call the fixture, `request.param` is not defined.

I updated the fixture to check if it has the attribute, otherwise, sets it to none and returns.